cplusplus/include/vips/VImage8.h

```diff
@@ -5784,6 +5784,7 @@ class VImage : public VObject {
 	 *   - **access** -- Required access pattern for this file, VipsAccess.
 	 *   - **fail_on** -- Error level to fail on, VipsFailOn.
 	 *   - **revalidate** -- Don't use a cached result for this operation, bool.
+	 *   - **rgb128** -- Enable 128-bit rendering
```

Comment:
These doc comments are generated automatically, so you don't need to include them in a PR. 

This text will be replaced by the `_("Enable 128-bit SVG rendering")` in the ARG declaration next time this script runs:

https://github.com/libvips/libvips/blob/master/cplusplus/gen-operators.py

(no need for you to do anything, just a note)